english/sing/something-else/sing-something-else.md

```diff
@@ -4,4 +4,6 @@ Aha, you say to yourself as you figure out the perfect song.
 
 Once again, you clear your throat and begin to sing Rick Astley's "Never Gonna Give You Up".
 
-As you sing, a smile grows on the monster's face and you both begin to dance.
\ No newline at end of file
+As you sing, a smile grows on the monster's face and you both begin to dance.
+
+His hand holding yours is not as soft as you expected. It is cold and slightly compressed; you can feel the grooves on his skin where his flesh shrunk and puckered. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.